Skip next work item

daszo opened the pull request:
I added the option to call the argument skip and it will finish the interval and go to the next
Closes #30

daszo committed:
added a skip function with a test to see if it works

daszo committed:
added skip to the help string

Andeskjerf commented:
Looks good! Was it necessary to change the condition for this to work though? There's also a printed help message in the README, it'd be nice to see the new operation there as well.

daszo commented:
Thanks! You are correct, I do not need the Config file at all! I'll change it when I have time again. I'll add the feature to the README as well.

daszo committed:
added a double click handler and updated instructions in the README

daszo committed:
reverted the conditions back to the origional state (removed the config condition)

daszo commented:
I reverted back to the original conditions and I added it to the README. i also added a double click handler bash file and instructions if people want to use that to skip.

Need to add a working test next

daszo committed:
added a working test function

daszo commented:
Added a working test function

daszo commented:
@Andeskjerf I think I updated it better this time

jolars commented on README.md:
```diff
@@ -38,6 +38,20 @@ Include the module in your bar and you're set!
+"custom/pomodoro": {
+	"format": "{}",
+	"return-type": "json",
+	"exec": "~/path/to/dobule-click-handler.sh",
```
Why not just use `on-double-click`?

jolars replied:
Ah, nevermind, I just saw:

> It's suggested not to have multiple configurations for the same mouse button. For example: on-click, on-double-click, on-triple-click are defined. When triple click is triggered the module will execute commands for on-click, on-double-click, on-triple-click sequentially because of Gdk provide such events.

jolars commented on README.md:
```diff
@@ -38,6 +38,20 @@ Include the module in your bar and you're set!
 
+## Double click skip setup
+
+If you want to add the functionality to skip, you can utilize the double-click-handler.sh
```
Maybe it would be better to provide a menu handler instead?

https://man.archlinux.org/man/extra/waybar/waybar-menu.5.en
